app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -675,6 +668,13 @@ export const ContentForm: React.FC<IContentFormProps> = ({
                       Unpublish
                     </Button>
                   </Show>
+                  <Button
```

Comment:
The Publish button needs to become the default main button.  Change the variant and type of both the Publish and the Save.

Reply:
Updated, thanks

Reply:
Unless I'm blind or you haven't pushed an update this isn't fixed.  

The Publish button needs to have these values.
`type=submit`
`variant=ButtonVariant.primary`

The Save button needs to have these values.
`variant=ButtonVariant.secondary`

Reply:
Updated, for the Publish button we don't need to set variant as it will be set to ButtonVariant.primary by default. Thanks